File: rust/src/models/boolean_entitlement_value.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct BooleanEntitlementValue {
    pub enabled: bool,
}

impl BooleanEntitlementValue {
    pub fn new(enabled: bool) -> Self {
        Self { enabled }
    }
}

File: typescript/src/models/booleanEntitlementValue.ts
// this file is @generated

export interface BooleanEntitlementValue {
  enabled: boolean;
}

export const BooleanEntitlementValueSerializer = {
  _fromJsonObject(object: any): BooleanEntitlementValue {
    return {
      enabled: object["enabled"],
    };
  },

  _toJsonObject(self: BooleanEntitlementValue): any {
    return {
      enabled: self.enabled,
    };
  },
};
